src/Game/Web/Requests/DNS.elm

```diff
@@ -74,110 +78,150 @@ errorMessage url str =
             fail "Unexpected error"
 
 
-encoder : String -> Value
-encoder url =
+encoder : String -> String -> Value
+encoder url networkId =
     Encode.object
-        [ ( "url", Encode.string url )
+        [ ( "network_id", Encode.string networkId )
+        , ( "address", Encode.string url )
         ]
 
 
 decoder : Url -> Decoder Response
 decoder url =
```

Comment:
This change became completely unnecessary, see [Helix#270](https://github.com/HackerExperience/Helix/issues/270l)